app/Console/Commands/MigrateOSCIPublicationOne.php

```diff
@@ -26,6 +29,107 @@ class MigrateOSCIPublicationOne extends Command
      */
     protected $description = 'Migrate an OSCI publication from a migration file to a website DigitalPublication';
 
+    /**
+     * configure $block (a Block model) with $data
+     *
+     */
+    private function configureFigBlock($figure,$block)
+    {
+        switch ($figure->figure_type) {
+            case 'html_figure':
+                // TODO: if $figure->html_content_src set embed_type => url and url => html_content_src
+
+                $block->type = 'media_embed';
+                $block->content = json_encode([
+                    "size" => "m",
+                    "embed_type" => "html",
+                    "embed_code" => $figure->html_content,
+                    "embed_height" => "400px",
+                    "disable_placeholder" => true
+                ]);
+
+                $block->save();
+
+                break;
+
+            // TODO: if layered_image instantiate a layered image block 
+            case 'layered_image':
+            case 'iip_asset':
+
+                $block->type = 'image';
```

Comment:
We want this block to be of type `layered_image_viewer`. And ideally each image would be added to a `layered_image_viewer_img` or `layered_image_viewer_overlay` repeater image field. Look here for more details on the block fields: https://github.com/art-institute-of-chicago/artic.edu/blob/feature/osci-migration-update/resources/views/admin/blocks/layered_image_viewer.blade.php

Reply:
This is set, minus the layer operations which are blocked by image block media attachments